LogicalTypes.md

```diff
@@ -219,7 +219,7 @@ decimal point, and the precision stores the maximum number of digits supported
 in the unscaled value.
 
 If not specified, the scale is 0. Scale must be zero or a positive integer less
-than the precision. Precision is required and must be a non-zero positive
+than or equal to the precision. Precision is required and must be a non-zero positive
```

Comment:
Thanks for the PR! Could you also add the comment to the thrift definition: https://github.com/apache/parquet-format/blob/master/src/main/thrift/parquet.thrift#L253 ?

Reply:
Will do.